### DISCUSSION TITLE
Conditional Execution in PyOptiSlang - Parameter-Based Program Execution

### DISCUSSION THREAD

**Discussion by Knam-Ansys** [ansys/pyoptislang#375] [Q&A] [answered]
@rfahlberg
Hi, thanks for your help, I successed to make prototype of optisalng simulation script.

I'm working with multiple AEDTs and want to create a code where different programs run based on specific parameters.
![image](https://github.com/user-attachments/assets/1d10e62b-a116-4662-b54f-4324da4f7224)

As shown in the attached images, I'm looking to implement conditional execution to manage program workflows. However, I can't find the specific functionality in PyOptiSlang.
![image](https://github.com/user-attachments/assets/41a2896b-ae9a-422d-921d-559cbe416668)

Could you confirm if PyOptiSlang currently supports this type of conditional execution?

**Comment by rfahlberg** [comment, marked as answer]
@Knam-Ansys This feature is currently not supported. We will add this in a future release. I created an according issue: https://github.com/ansys/pyoptislang/issues/378.

**Reply by Knam-Ansys** [reply]
Thank you for the update and for creating the issue. Looking forward to the future release! 🚀
